src/main.css

```
@@ -64,4 +64,7 @@ body {
 }
 .hover-card:hover {
    transform: scale(1.07);
+}
+.archived {
+  background-color:#cccccc
 }
```

Add a new line here.
```suggestion
}

```